src/helpers/zoom.js

```diff
@@ -1,65 +1,145 @@
 import core from 'core';
 import { stepToZoomFactorRangesMap, getMaxZoomLevel, getMinZoomLevel } from 'constants/zoomFactors';
 
-export const zoomIn = () => {
-  const currentZoomFactor = core.getZoom();
-  if (currentZoomFactor === getMaxZoomLevel()) {
-    return;
-  }
-
-  const step = getStep(currentZoomFactor);
-  const newZoomFactor = currentZoomFactor + step;
-  zoomTo(Math.min(newZoomFactor, getMaxZoomLevel()));
-};
-
-export const zoomOut = () => {
-  const currentZoomFactor = core.getZoom();
-  if (currentZoomFactor === getMinZoomLevel()) {
-    return;
-  }
-
-  const step = getStep(currentZoomFactor);
-  const newZoomFactor = currentZoomFactor - step;
-  zoomTo(Math.max(newZoomFactor, getMinZoomLevel()));
-};
-
-const getStep = currentZoomFactor => {
+function getStep(currentZoomFactor) {
```

Comment:
It looks like the convention of declaring a function is slightly different from the previous code.

I observe  other .js files in this helpers folder follow different format as well.
i.e cssVariablesParser.js vs. acessibility.js

@ZhijieZhang I think we should have a consistent convention?

Reply:
I think we could consider going back using traditional functions in most cases. Arrow functions are anonymous functions and makes debugging a lot harder. See for example this performance analysis: 
<img width="277" alt="Screen Shot 2020-07-17 at 9 06 53 AM" src="https://user-images.githubusercontent.com/2638877/87807678-67cf7280-c80d-11ea-84b9-2637a4e9902f.png">

Would be so much more informative if we would see function names rather than 3 levels of anonymous. 

There are certain cases though where it is okay to use arrow functions like if they are class properties or used in some of the callbacks like `map(()=>{})`. In cases like `getStep` there is no benefit of using arrow function (like saving this binding) instead of normal function definition, only downside of anonymous function.

Reply:
@jniemin  Interesting. Thanks for pointing that out.
I'm fine with that.

@ZhijieZhang what are your thoughts?

Reply:
I think both styles can exist in the codebase since both are valid JavaScript and in the end, Babel will just transpile arrow functions to normal functions.

When I started working on this project I think we were already be using arrow functions in many places. So I just kept using this, even for functional components. I think I like how arrow functions visually make the code a bit clearer to read, but Jussi makes a good point here about normal(plain?) functions. Given that a lot of the places are using arrow functions, I don't think we should force using `function` at this moment since they are many places to modify. But I don't think we should disallow the usage of `function` either.

I just used to performance analyzer for the UI and it seems that if we use `.map(() => {...})` then it will show up as an `(anonymous)`. If our goal is to make the debugging experience very well, then I guess eventually we want to use `.map(function funcName() {}.bind(..))` everywhere in the code? This feels more verbose than just an arrow function :(

Also I tried in Babel playground and it seems that arrow functions will be transpiled to a function with a name. Not 100% sure but does this mean using arrow functions won't actually make the code harder to debug?